### PR TITLE
Added a way to change image styling through blueprint and css

### PR DESCRIPTION
**Pull request by aashishvasu**
Hi Jason, I've added a way for people to choose the way they want their image to appear in the Intro section. Thought it would be a valuable addition to the main theme. Pls refer to images below:

None
Circle
Rounded

### CODE REVIEW

File: blueprints/modular/intro.yaml [hunk 1]
```diff
@@ -59,6 +59,16 @@ form:
                   validate:
                     type: textarea
 
+                header.img_style:
+                  type: select
+                  label: Image Style
+                  description: Choose if the image should be a circle or have rounded corners.
+                  size: medium
+                  options:
+                    '': None
```
Comment (jasonccox) [inline blueprints/modular/intro.yaml]: Let's rename this to make it a bit more obvious what no style means.
```suggestion
                    '': Square
```

File: blueprints/modular/intro.yaml [hunk 2]
```diff
@@ -59,6 +59,16 @@ form:
                   validate:
                     type: textarea
 
+                header.img_style:
+                  type: select
+                  label: Image Style
+                  description: Choose if the image should be a circle or have rounded corners.
```
Comment (jasonccox) [inline blueprints/modular/intro.yaml]:
```suggestion
                  description: Choose the shape of the image.
```

File: scss/intro.scss [hunk 3]
```diff
@@ -11,6 +11,14 @@
 	width: 50%;
 }
 
+.intro__img.rounded {
```
Comment (jasonccox) [inline scss/intro.scss]: Can we nest these new selectors in the `.intro__img` selector above
```scss
.intro__img {
    ...
    &.rounded {
        ...
    }
    ...
}

File: scss/intro.scss [hunk 4]
```diff
@@ -11,6 +11,14 @@
...
+	border-radius: 20px;
```
Comment (jasonccox) [inline scss/intro.scss]: Most of the theme uses `rem` units instead of pixels. Let's do the same here.

**Comment by jasonccox** [comment]
Thanks again for all the work you're doing on this!

**Comment by aashishvasu** [comment]
Hey Jason, I havent forgotten about these commits, just gotten awfully busy with work. I will include these as soon as I can over the coming weeks!

**Comment by jasonccox** [comment]
Okay, no worries!

**Comment by aashishvasu** [comment]
Hi Jason, 

Long time!!
I finally have some free time to work on the PR, so here is the first one. I have incorporated your changes already, just let me know if it works for you!

**Comment by jasonccox** [comment]
Hi Aashish,

I'm excited to see that you've done some more work on these pull requests. Unfortunately I'm pretty busy as a full-time student right now, so it might be a bit before I am able to take a look at them again. I'll do my best to review them soon!

File: scss/intro.scss [hunk 5]
```diff
@@ -9,6 +9,14 @@
 .intro__img {
 	flex: 0 100%;
 	width: 50%;
+
+	&.rounded {
+		border-radius: 3.5rem;
```
Comment (jasonccox) [inline scss/intro.scss]: How about using a percent here as well so that it looks similar no matter the image's size (for example, on a phone).

Comment (aashishvasu) [reply scss/intro.scss]: This has been changed from rem notation to percentage.

**Comment by jasonccox** [comment]
I had to make a few changes to get this working -- most notably, using string concatenation in `intro.html.twig` instead of interpolation (since the relevant string was already an interpolation). As a result, I just squashed and merged the changes myself in 309f766ff6d35d94884fa6d41dd375655a7025b4.

**Pull request closed without merging**